tests/test_main.py

```diff
@@ -229,19 +225,6 @@ def test_resume_env_var(tmp_dir, monkeypatch):
     assert Live()._resume
 
 
-@pytest.mark.parametrize("metric", ["m1", os.path.join("train", "m1")])
```

Comment:
minor: Would be nice to have an explicit test overriding a value

Reply:
Added, thanks!